code/track.ss

```diff
@@ -27,27 +56,27 @@
 ;; as this one. Makefile helps ensure this.
 (define (get-problem-specification problem)
   (let* ((problem-dir (format "../problem-specifications/exercises/~a" problem))
-	 (spec (directory-list problem-dir)))
+         (spec (directory-list problem-dir)))
```

Comment:
Most of this is from me reformatting the files with my emacs.  I think it is related to the issue you were talking about having with exercism.  I don't know what the difference is between our emacs setup0s that makes this happen, but when I get files you have worked on the formatting is all messed up, so I have to reformat it even though the formatting doesn't look wierd on github :confused: